app/assets/javascripts/html5_webshim.js

```diff
@@ -0,0 +1,15 @@
+$(document).on('ready page:load', function(){
+    window.webshim.setOptions('forms-ext', {
+    lazyCustomMessages: true,
+    replaceUI: true,
+    waitReady: true,
+    type: 'date',
+    date: {
+      startView: 2,
+      openOnFocus: true
+    }
+  });
+
+  window.webshim.polyfill('forms-ext');
+  window.onunload = function(){};
```

Comment:
The pollyfill does not work if you navigate away from the page and back again. This fixes it somehow.. I think it has something to do with the bfcache. I forgot where I came across it, maybe it was this http://stackoverflow.com/questions/158319/cross-browser-onload-event-and-the-back-button